runtime/ast/import_test.go

```diff
@@ -86,6 +86,30 @@ func TestAddressLocation_MarshalJSON(t *testing.T) {
 	)
 }
 
+func TestAddressContractLocation_MarshalJSON(t *testing.T) {
```

Comment:
This isn't related to this PR right?

Reply:
Good catch! No, this test doesn't belong to this PR, I accidentally committed it. I'll remove